docker-compose.yml

```diff
@@ -65,7 +65,7 @@ services:
       - 8984:8984
 
   editor-client:
-    image: liberoadmin/editor-client:${TAG-latest}
+    image: libero/editor-client:local
```

Comment:
This looks breaking to me, as in if I was to do a fresh clone of the repo and then run `docker-compose up` it would fail as there isn't a container published under the `libero` name, nor a published local tag.